src/Basho/Riak/Object.php

```diff
@@ -225,19 +235,20 @@ public function addLink($obj, $tag = null)
      * @param mixed $obj - Either a Object or a Link object.
      * @param string $tag - Optional link tag. (default is bucket name,
      * ignored if $obj is a Link object.)
+     *
      * @return $this
      */
     public function removeLink($obj, $tag = null)
     {
         if ($obj instanceof Link) {
-            $oldlink = $obj;
+            $old_link = $obj;
         } else {
-            $oldlink = new Link($obj->bucket->name, $obj->key, $tag);
+            $old_link = new Link($obj->bucket->name, $obj->key, $tag);
```

Comment:
Good point, but PhpStorm was complaining about spelling, I just wanted to shut it up. I am not to worried about this var since its internal scope to that method, but I will updated it to camelcase next time i am in there.